### DISCUSSION TITLE
ScriptReference/CursorMode.ForceSoftware

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#693] [Notes]
# ScriptReference/CursorMode.ForceSoftware

https://docs.unity3d.com/ScriptReference/CursorMode.ForceSoftware.html

**Comment by vertxxyz** [comment]
This API is incompatible with the new input system by design. See:
[Issue Tracker - Cursor does not move when using CursorMode.ForceSoftware in New Input System](https://issuetracker.unity3d.com/issues/cursor-does-not-move-in-the-game-view-when-using-cursormode-dot-forcesoftware-in-new-input-system).
